.travis.yml

```diff
@@ -2,4 +2,3 @@ language: node_js
 node_js:
   - '10'
   - '8'
-  - '6'
```

Comment:
I added some async / await and see no reason to maintain support for non-async-await versions of Node.js. This is, after all, a CLI aimed at open source maintainers.